DESCRIPTION

```diff
@@ -23,9 +23,13 @@ Depends:
     R (>= 4.3),
     methods,
     progress,
-    R6
+    R6,
+Imports: 
+    cli
 Suggests: 
-    testthat (>= 2.1.0)
+    testthat (>= 2.1.0),
+    DSLite, 
```

Comment:
That makes a circular dependency and the CRAN will not accept that.

Reply:
I think so long as it is in the `suggests` and not the `depends` field it should be ok? But I can remove the tests if you prefer.

Reply:
Yes please move the tests to DSLite.